src/plugins/js-eval/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM node:slim
-RUN npm i --prefix /run date-fns airbnb-js-shims full-icu
+RUN npm i --prefix /run airbnb-js-shims full-icu acorn nearley
+# acorn nearley are needed by engine262
+RUN mkdir /run/vendor && curl https://engine262.js.org/engine262.js > /run/vendor/engine262.js
```

Comment:
I want the download to happen ahead of time, and have the file checked into git. So `jellobot-repo/vendor/engine262.js` would be written by the script, and the Dockerfile should just `ADD` the file.

Reply:
Yea sry, done